src/components/Settings/index.jsx

```diff
@@ -93,8 +113,14 @@ export class Settings extends Component {
           diskUsage: diskUsageQuery,
           instance: instanceQuery
         }
+        shouldDisplayViewOfferButton =
+          instanceModel.shouldDisplayOffers(data) || hasAnOffer(data)
```

Comment:
Are there still cases where we *don't* display the button?

Reply:
Yep. If cozy-client version is a bit late